src/gl/pointFeature.js

```diff
@@ -318,6 +319,20 @@ geo.gl.pointFeature = function (arg) {
 
   ////////////////////////////////////////////////////////////////////////////
   /**
+   * Return mapper where actor gets it behavior and data
+   *
+   * @returns {vgl.mapper}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.mapper = function () {
```

Comment:
Its probably better if we return actors (as then we can move that API in a common GL class). This would be because a feature like graph may have more than one actor. Also, in the future it is possible that point feature will split the data into many actors (for performance reasons). 

this.actors = function() {
return [m_actor]; 
}

What you think of it? 

Thanks for pushing these changes.